x-pack/plugins/siem/public/components/tables/helpers.tsx

```diff
@@ -87,7 +87,12 @@ export const getRowItemDraggables = ({
       const id = escapeDataProviderId(`${idPrefix}-${attrName}-${rowItem}`);
       return (
         <React.Fragment key={id}>
-          {index !== 0 ? <>,&nbsp;</> : null}
+          {index !== 0 ? (
```

Comment:
Optional, can be simplified a teeny bit:
```
{index !== 0 && (
  <>
    {', '}
    <Spacer />
  </>
)}
```

Reply:
Thanks! Will do